suite-common/formatters/src/tests/cryptoAmountFormatter.test.ts

```diff
@@ -0,0 +1,42 @@
+import { prepareCryptoAmountFormatter } from '../formatters/prepareCryptoAmountFormatter';
+
+const CryptoAmountFormatter = prepareCryptoAmountFormatter({
+    coins: [],
+    locale: 'en',
+    bitcoinAmountUnit: 0,
```

Comment:
What is this `0`? Shouldn't there be some enum?

Reply:
I'll change it to `PROTO.AmountUnit.BITCOIN` which is essentially 0